release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -85,8 +85,7 @@ For more information, see xref:../installing/installing_aws/installing-aws-netwo
 
 With this update you can install {product-title} to an existing VPC with installer-provisioned infrastructure, extending the worker nodes to Local Zones subnets. The installation program will provision worker nodes on the edge of the AWS network that are specifically designated for user applications by using NoSchedule taints. Applications deployed on the Local Zones locations deliver low latency for end users. 
 
-// Link commented out pending release of details on AWS Local Zone. All approvals received.
-// For more information, see .../installing/installing_aws/installing-aws-localzone.adoc[Local Zone customizations for installation on AWS VPC subnets].
+For more information, see xref:../installing/installing_aws/installing-aws-localzone.adoc[Local Zone customizations for installation on AWS VPC subnets].
```

Comment:
@GroceryBoyJr, please use the final title:
```suggestion
For more information, see xref:../installing/installing_aws/installing-aws-localzone.adoc[Installing a cluster using AWS Local Zones].
```

Reply:
Perfect, thank you @kalexand-rh